sweepbatcher/sweep_batch.go

```diff
@@ -830,7 +830,7 @@ func (b *batch) publishBatchCoop(ctx context.Context) (btcutil.Amount,
 
 	totalWeight := int64(weightEstimate.Weight())
 
-	fee = b.rbfCache.FeeRate.FeeForWeight(totalWeight)
+	fee = b.rbfCache.FeeRate.FeeForWeight(lntypes.WeightUnit(totalWeight))
```

Comment:
Conversion to `int64` can be skipped here and in similar cases above.

```go
totalWeight := lntypes.WeightUnit(weightEstimate.Weight())

fee = b.rbfCache.FeeRate.FeeForWeight(totalWeight)
```

Reply:
Or just `fee = b.rbfCache.FeeRate.FeeForWeight(weightEstimate.Weight())` directly. Same above.